main.tf

```diff
@@ -49,6 +50,7 @@ resource "aws_security_group" "default" {
     from_port       = 0
     to_port         = 0
     protocol        = -1
+    description     = "Allow access from security group"
```

Comment:
@syphernl since we're going to change this, I think a more descriptive label may be "Allow ingress to groups listed in `var.ingress_security groups`"

Reply:
I have changed the labels for both securitygroups to be more descriptive.